keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/ResourceRepositoryTests.kt

```diff
@@ -213,10 +215,10 @@ abstract class ResourceRepositoryTests<T : ResourceRepository> : JUnit5Minutests
         test("the resource is no longer returned when listing all resources") {
           subject.allResources(callback)
 
-          verifyZeroInteractions(callback)
+          verify(callback, never()).invoke(eq(ResourceHeader(resource)))
```

Comment:
Not clear how this ever worked. Investigating. I think I might not be resetting mocks correctly.